src/cmake/cmakeExecutable.ts

```diff
@@ -66,5 +67,5 @@ export async function getCMakeExecutableInformation(path: string): Promise<CMake
 }
 
 export async function setCMakeDebuggerAvailableContext(value: boolean): Promise<void> {
-    await util.setContextValue("vscode-cmake-tools.cmakeDebuggerAvailable", value);
+    await setContextAndStore("vscode-cmake-tools.cmakeDebuggerAvailable", value);
```

Comment:
This is a nit, as it's not something that you actually made changes to, but I think it might be nice if we change the usage of `vscode-cmake-tools.cmakeDebuggerAvailable`, to match the pattern everywhere else `cmake:cmakeDebuggerAvailable`.

Reply:
refactored!